crate_universe/src/metadata/dependency.rs

```diff
@@ -92,6 +93,11 @@ impl DependencySet {
         // https://doc.rust-lang.org/cargo/reference/build-scripts.html#the-links-manifest-key
         // https://doc.rust-lang.org/cargo/reference/build-scripts.html#-sys-packages
         // https://doc.rust-lang.org/cargo/reference/build-script-examples.html#using-another-sys-crate
+
+        // TODO: The above/below is incorrect fyi. It is actually only when any
```

Comment:
I agree! Could I bother you to rewrite the comment and filter logic accordingly?

Reply:
I'm not interesting in doing that as we don't use crate_universe internally at all (we only use the rules_rust part). I was just trying to not break existing functionality.

Reply:
Actually I will see if I can quickly do this on Monday, or Tuesday, but no promises haha.

Reply:
Done.